carbon-design-system/carbon
# fix(colors): update classic sass file variable initialization

Closes #8565

This updates our color build step to emit all variables before the mixins in the file.

#### Changelog

**New**

**Changed**

- Update the colors build step to emit all variables before the mixins in the file

**Removed**

#### Testing / Reviewing

- Pull down the PR and run `yarn build` in `packages/colors`
- Inspect `scss/_mixins.scss`
- Verify that all color variables are declared at the start of the file that are included in the `ibm--colors` and `carbon--colors` mixins
- Verify that all variables defined in the mixins use the `!global` flag
